src/connectors/freemusicarchive.js

```diff
@@ -54,14 +54,15 @@ function setupCommonProps() {
 	Connector.pauseButtonSelector = '.playbtn-paused';
 
 	Connector.getUniqueID = () => {
-		const playingItem = document.querySelector('.gcol-electronic');
+		const playingItem = document.querySelector('.play-item.gcol-electronic');
 		if (!playingItem) {
 			return null;
 		}
 
 		const match = /(tid-\d+)/.exec(playingItem.className);
 		if (match) {
-			return match[0];
+			const idString = match[0].split('-');
```

Comment:
Is this change even required? 

> The identifier does not  have to be in any specific format. The uniqueness is only needed within  the scope of the connector

Reply:
I think we need to provide the correct value for the connector in case we need it anytime in the future. FMA uses numeric IDs so the returned value should be numeric ID. I think the reason behind the prefixing is the naming rule for [CSS selectors](https://www.w3.org/TR/CSS22/syndata.html#characters):
> they cannot start with a digit, or a hyphen followed by a digit.

Reply:
It's not required, but omitting `tid` prefix would make the connector more robust and consistent in cases where unique IDs will have no prefix, or other prefixed.

The only drawback is loosing edited tracks, but I'm not sure if this a real issue. FMA connector was broken for a while, but no one reported.